src/Navigation/Education/index.css

```diff
@@ -0,0 +1,133 @@
+.education {
+  padding: var(--section-padding);
+  font-family: var(--project-font-family);
```

Comment:
We are not changing the fonts for the whole project, so we can just give fonts to the body and that's it!
We will not need to apply fonts for each section.